static/js/app.js

```diff
@@ -321,6 +321,19 @@ function reportClientError(level, code, client_error) {
   });
 }
 
+window.onerror = function reportClientException() {
+
+  $.ajax({
+    type: 'POST',
+    url: '/client_exception',
+    data: JSON.stringify({
+      error: [].slice.call(arguments)
```

Comment:
This data type is going to be an array. Is that desirable?

The arguments to `window.onerror`s handler are probably pretty well-defined, so we can be more specific than this.

Reply:
Good point; the arguments are different in old browsers that we're not supporting anyway. I'll change it to be more specific.